installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -98,8 +98,16 @@ include::modules/installation-aws-tested-machine-types.adoc[leveloffset=+2]
 * See link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation for more information about AWS Local Zones and the supported instances types and services. 
 
 include::modules/installation-generate-aws-user-infra-install-config.adoc[leveloffset=+2]
+// Suggest to standarize edge-pool's specific files with same prefixes, like: machine-edge-pool-[...] or compute-edge-pool-[...] (which is more compatible with install-config.yaml/compute)
 include::modules/machines-edge-machine-pool.adoc[leveloffset=+2]
 include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[Changing the MTU for the cluster network]
+* xref:../../networking/changing-cluster-network-mtu.adoc#nw-ovn-ipsec-enable_configuring-ipsec-ovn[Enabling IPsec encryption]
```

Comment:
those references should be aligned with the MTU declaration

Reply:
Aligned as in level in the document, content, or grammar?

Reply:
content. The reference of MTU might be done after the MTU limitations, and before cluster creation as it requires manual actions.

Reply:
Sure. Want to attempt this here, or in https://github.com/openshift/openshift-docs/pull/57427?

Reply:
let's do it here, if you're fine. My comment above is more information, the change is already done :)